Documentation/devel/quickstart-dev.md

```diff
@@ -0,0 +1,97 @@
+# Preparing development environment and first rkt build
+This is an example configuration for fresh installation of Ubuntu 16.04 GNOME.
+This doc is also short quick start guide. For more accurate click [here](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md)
+### Required software
+On fresh system install curl and git:
+```sh
+$ sudo apt-get install git curl
+```
```

Comment:
There are now two "Required software" sections.
`install-deps-debian-sid.sh` already installs git and wget. You could make the examples just use wget instead of curl, so that you can just replace this section with the other section below.